src/liboslexec/backendllvm.h

```diff
@@ -376,15 +376,24 @@ class BackendLLVM : public OSOProcessorBase {
     /// is true, pass pointers even for floats if they have derivs.
     /// Return an llvm::Value* corresponding to the return value of the
     /// function, if any.
-    llvm::Value *llvm_call_function (const char *name,  const Symbol **args,
-                                     int nargs, bool deriv_ptrs=false);
-    llvm::Value *llvm_call_function (const char *name, const Symbol &A,
+    llvm::Value *llvm_call_function (const char *name, OIIO::cspan<const Symbol *> args,
                                      bool deriv_ptrs=false);
     llvm::Value *llvm_call_function (const char *name, const Symbol &A,
-                                     const Symbol &B, bool deriv_ptrs=false);
+                                     bool deriv_ptrs=false) {
+        const Symbol *args[] = { &A };
+        return llvm_call_function (name, args, deriv_ptrs);
+    }
+    llvm::Value *llvm_call_function (const char *name, const Symbol &A,
+                                     const Symbol &B, bool deriv_ptrs=false) {
+        const Symbol *args[] = { &A, &B };
+        return llvm_call_function (name, args, deriv_ptrs);
```

Comment:
Just for fun, I think that this should also work:

    return llvm_call_function (name, { &A, &B }, deriv_ptrs);

yes?

Reply:
It does! And at that point, there isn't much need to add these overloads at all.

Reply:
I'll leave the overloads for the time being since they are just 1 line and save  a bunch of code.

For the `cspan<Symbol*>` case, the fact that the overloads take a `Symbol&` make the client code a smidge cleaner.